js/src/modal.js

```diff
@@ -416,10 +416,23 @@ class Modal {
         return
       }
 
+      const isModalOverflowing = this._element.scrollHeight > document.documentElement.clientHeight
```

Comment:
woudn't it be better to use the window height instead of the html element?

Reply:
I copied this code from here without thinking deeply: 
https://github.com/twbs/bootstrap/blob/ca51b1eefd1d873dcfb8dcd11cf30af051456b11/js/src/modal.js#L448-L450

Reply:
LGTM this way because we are taking care of the client style